bears/typescript/TSLintBear.py

```diff
@@ -26,16 +28,128 @@ class TSLintBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file,
-                         tslint_config: path="", rules_dir: path=""):
+                         tslint_config: path="", rules_dir: path="",
+                         directive_selectors_naming_convention:
+                             list=[True, "camelCase"],
+                         component_selectors_naming_convention:
+                             list=[True, "kebab-case"],
```

Comment:
didn't we just agree on another PR that we want to use the same convention all over coala?